sdk/commerce/arm-commerce-profile-2020-09-01-hybrid/LICENSE.txt

```diff
@@ -1,21 +0,0 @@
-The MIT License (MIT)
```

Comment:
@kazrael2119 could you confirm if it's changed into LICENSE and we fail to change this one?

Reply:
@qiaozha we weren't including this file in the npm package, only LICENSE, so it's not necessary to have two license files in the project.